src/app/components/Brand/index.jsx

```diff
@@ -31,6 +33,9 @@ const StyledGridWrapper = styled.div`
   background-color: ${C_POSTBOX};
   height: ${BANNER_HEIGHT};
   width: 100%;
+  @media (min-width: ${group3ScreenWidthMax}) {
+    padding: 0 ${GEL_SPACING_DBL};
```

Comment:
Due to the implementation detail described in the comments of this PR, the Brand component suffers a visual regression above 1007px due to it previously relying on padding on the grid wrapper. As the padding has been removed above 1007px (see implementation detail for explanation) we need to add the padding on the Brand component. 

This is a temporary solution until the [grid is completely removed from the branch component](https://github.com/BBC-News/simorgh/issues/827). 

NB: If others feel like the regression is not enough for this change to be included in this work I'm happy to open a bug ticket and fix forward in a follow-up PR.